<launch>
  <arg name="kinect_name" default="camera"/>

  <node name="kinects_merge" pkg="kinects_human_tracking" type="merge_kinects_pc" output="screen">
    <param name="first_topic_name" value="kinect1/background_sub" />
    <param name="second_topic_name" value="kinect2/background_sub" />
    <param name="out_frame" value="world" />
    <param name="out_topic_name" value="kinect_both" />
  </node>
</launch>
